### DISCUSSION TITLE
use Page object in a method's signature throws

### DISCUSSION THREAD

**Discussion by maayansharon10** [jsvine/pdfplumber#770] [Q&A] [answered]
Hi,
Newbie question here :)
I have this piece of code : 

```
import pdfplumber
def contains_rects(page_num, page):
  return len(page.rects) > 3 

```
I want to change the method signature to be more informative, but I cannot seem to specify Page as an object, for example - 
`def contains_rects_curves_tables(page_num: int, page: pdfplumber.Page) -> int:
`
but "pdfplumber.Page" throws.

also tried -
` from pdfplumber import Page
def contains_rects_curves_tables(page_num: int, page: Page) -> int:
`
but then the import line throws 
`ImportError: cannot import name 'Page' from 'pdfplumber' (/usr/local/lib/python3.7/dist-packages/pdfplumber/__init__.py)`

Any suggestions? 

Thank you!

**Comment by samkit-jain** [comment, marked as answer]
Hi @maayansharon10 Appreciate your interest in the library. The reason is because the `Page` class is not exposed. However, if you want to still import it and use as a type hint, you can use `pdfplumber.page.Page` instead of `pdfplumber.Page`. Notice the `.page.` import

**Reply by maayansharon10** [reply]
Thank you @samkit-jain ! Appreciate the explanation, `from pdfplumber.page import Page` is just what I've been looking for 👍
